addons/spreadsheet/static/src/helpers/model.js

```diff
@@ -76,9 +77,49 @@ export async function freezeOdooData(model) {
             }
         }
     }
+    if (model.getters.getGlobalFilters().length === 0) {
+        return data;
+    }
+    data.sheets.push(exportGlobalFiltersToSheet(model, data));
     return data;
 }
 
+function exportGlobalFiltersToSheet(model, data) {
```

Comment:
relational filters doesn't seem to be exported correctly (an empty string is exported instead of the records display name)
```
{
    "A1": {
        "content": "Filter",
        "style": 2
    },
    "B1": {
        "content": "Value",
        "style": 2
    },
    "A2": {
        "content": "Period"
    },
    "B2": {
        "content": "Last 30 Days"
    },
    "A3": {
        "content": "Stage"
    },
    "B3": {
        "content": ""
    },
    "A4": {
        "content": "Salesperson"
    },
    "B4": {
        "content": ""
    },
    "A5": {
        "content": "Sales Team"
    },
    "B5": {
        "content": ""
    },
    "A6": {
        "content": "Country"
    },
    "B6": {
        "content": ""
    },
    "A7": {
        "content": "City"
    },
    "B7": {
        "content": ""
    },
    "A8": {
        "content": "Medium"
    },
    "B8": {
        "content": ""
    },
    "A9": {
        "content": "Source"
    },
    "B9": {
        "content": ""
    }
}
```